### PR TITLE
update instructions in readme

### PR DESCRIPTION
**Pull request by Akuli**
@ThePhilgrim Can you review this?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,22 +136,29 @@ Of course, just install the tetris plugin. See [more_plugins](more_plugins/).
...
+    but that's not required.
+- Don't worry too much about whether your pull code is good or not.
+    I review the pull requests, and there's checks running on GitHub Actions.
+- You can talk with me on GitHub issues,
+    or chat at [##learnpython on libera](https://kiwiirc.com/nextclient/irc.libera.chat/##learnpython).
+    I am on ##learnpython at about 3PM to 9PM UTC.
+    Don't worry about asking too many questions.
+    I'm not easily annoyed by that,
+    and if you create several pull requests after I answer all your questions,
+    I think answering the questions was definitely worth it.
```
Comment (ThePhilgrim) [inline README.md:158]:
```suggestion
    I appreciate your contributions!
```

**Review by ThePhilgrim — CHANGES_REQUESTED** [review]
Please note that these are just suggestions. Feel free to implement what you like and discard the rest.

File: README.md [hunk 2]
```diff
@@ -136,22 +136,29 @@ Of course, just install the tetris plugin. See [more_plugins](more_plugins/).
...
-If you want to develop porcupine, make a fork of Porcupine
-with the button in the top right corner of this page.
-Then install Python 3.7 or newer and
-[git](https://git-scm.com/), and run these commands:
+If you want to do something to Porcupine, that's awesome!
+I have tried to make contributing easy:
+- There is not much boilerplate involved in the contributing process.
+    You just create a pull request and that's it.
+    You can choose an issue and just start working on it;
+    you don't have to ask me to assign the issue to you or anything like that.
```
[the remark this answers is not in the document]
Comment (ThePhilgrim) [reply README.md]: I see that this was changed in a second commit. Feel free to use whichever version you prefer.

File: README.md [hunk 3]
```diff
@@ -136,22 +136,29 @@ Of course, just install the tetris plugin. See [more_plugins](more_plugins/).
...
+- You don't need to read anything before you can get started.
+    I recommend having a look at [the plugin API docs](https://akuli.github.io/porcupine/),
+    but that's not required.
+- Don't worry too much about whether your pull code is good or not.
+    I review the pull requests, and there's checks running on GitHub Actions.
+- You can talk with me on GitHub issues,
+    or chat at [##learnpython on libera](https://kiwiirc.com/nextclient/irc.libera.chat/##learnpython).
+    I am on ##learnpython at about 3PM to 9PM UTC.
+    Don't worry about asking too many questions.
+    I'm not easily annoyed by that,
```
[the remark this answers is not in the document]
Comment (Akuli) [reply README.md:156]: I'll keep and the next one this as they are, because I want to emphasize that asking questions isn't annoying.

**Review by ThePhilgrim — APPROVED** [review]

**Pull request merged**